catalystwan/endpoints/configuration_device_inventory.py

```diff
@@ -172,7 +172,7 @@ class ConfigType(str, Enum):
 
 class GenerateBoostrapConfigurationQueryParams(BaseModel):
     configtype: Optional[ConfigType] = Field(default=ConfigType.CLOUDINIT)
-    incl_def_root_cert: Optional[bool] = Field(default=False)
+    incl_def_root_cert: Optional[bool] = Field(default=False, alias="inclDefRootCert")
```

Comment:
`BaseModel` comes from pydantic v2, therefore please use `validation_alias` and `serialization_alias`.